src/Markdig/Parsers/ParagraphBlockParser.cs

```diff
@@ -176,6 +176,11 @@ private static char GetHeadingChar(ref StringSlice line, ref int count)
                 return headingChar;
             }
 
+            if (!line.CurrentChar.IsSpaceOrTab())
```

Comment:
I fixed it with 000393f46a3e8ed4bac5a685fdafa48c0dfb1a21, could you remove this change from this PR? Thanks!